lib/batchVersions.js

```diff
@@ -57,6 +57,9 @@ bedrock.events.on('bedrock-mongodb.ready', async () => {
 });
 
 export async function create({id, tokenizerId, options}) {
+  if(id && !(Number.isInteger(id) && id >= 0)) {
+    throw new TypeError('"id" must be a non-negative integer.');
+  }
```

Comment:
If you move this below the next conditional as an `else if`, you can remove the extra existence check on `id`.